sdk/core/azure-core/azure/core/configuration.py

```diff
@@ -88,16 +91,17 @@ def __init__(self, **kwargs):
         self.polling_interval = kwargs.get("polling_interval", 30)
 
 
-class ConnectionConfiguration(object):
+class ConnectionConfiguration:
     """HTTP transport connection configuration settings.
 
     Common properties that can be configured on all transports. Found in the
     Configuration object.
 
     :keyword int connection_timeout: A single float in seconds for the connection timeout. Defaults to 300 seconds.
```

Comment:
```suggestion
    :keyword int or float connection_timeout: A single value in seconds for the connection timeout. Defaults to 300 seconds.
```

Reply:
I'll move it to float following @kristapratico suggestion